EleksTubeHAX_pio/src/GLOBAL_DEFINES.h

```diff
@@ -75,14 +75,15 @@
 #define HOURS_TENS_MAP   (0x01 << HOURS_TENS)
 
 // Define the activate and deactivate state for the diplay power transistor
+// also define, how the dimming value is calculated
 #ifndef HARDWARE_IPSTUBE_CLOCK
   #define ACTIVATEDISPLAYS      HIGH    // Activate is HIGH for the IPSTUBEs
   #define DEACTIVATEDISPLAYS    LOW     // Deactivate is LOW for the IPSTUBEs
-  #define CALCDIMVALUE(x)       (x)
+  #define CALCDIMVALUE(x)       (x)     // Dimming value is directly used for software dimming
```

Comment:
Actually, CALCDIMVALUE is not used for software dimming.
(See `alphaBlend(dimming, color, TFT_BLACK);`)

This macros is defined based on activate displays - high and deactivate - low in this case, if there are some non-IPSTUBE clocks that can be PWMed with TFT_ENABLE pin in future.

Reply:
Gotit! I changed the comments in another branch which I will merge soon.